Simplenote/Classes/OptionsViewController.swift

```diff
@@ -371,8 +371,23 @@ private extension OptionsViewController {
 
     @IBAction
     func publishWasPressed(_ newState: Bool) {
-        SPTracker.trackEditorNotePublishEnabled(newState)
```

Comment:
Nipticky: Mind moving this invocation back to OptionsViewController?

Reasoning is:
- PublishController's sole responsibility is to encapsulate all the code related to publishing
- And symmetry! all of the other actions call SPTracker right here (ie.: `copyInterlinkWasPressed`)